cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -202,16 +202,15 @@ describe("Timeline", () => {
             // 3. Alignment of expanded GELS and placeholder of deleted message
             // 4. Alignment of expanded GELS, placeholder of deleted message, and emote
 
-            // Exclude timestamp from snapshot
-            const percyCSS =
-                ".mx_RoomView_body .mx_EventTile_info .mx_MessageTimestamp " + "{ visibility: hidden !important; }";
+            // Exclude timestamp from snapshot of mx_MainSplit
+            const percyCSS = ".mx_MainSplit .mx_MessageTimestamp { visibility: hidden !important; }";
```

Comment:
This should hide timestamps of not only an info event line but also an emote (and a normal event line for messages).